app/presenters/queries/content_item_presenter.rb

```diff
@@ -34,22 +34,31 @@ def initialize(scope, params = {})
         self.order = params[:order]
         self.limit = params[:limit]
         self.offset = params[:offset]
+        self.search_query = params[:search_query]
       end
 
       def present_many
-        scope = ::Queries::GetLatest.call(self.scope)
-        scope = join_supporting_objects(scope)
-        scope = order_and_paginate(scope)
-
-        extract_fields(scope)
+        parsing_enumerator(evaluate_query(ordered_extracted_fields))
       end
```

Comment:
Yes I suppose this particular method is a little tricky to parse inside out. I was trying to expose the ordering of these methods, but have done a bit more refactoring in f997d4297f23157e18fcab20c13d6a1f827b1a4e to try and express the intent more clearly. 

I guess from a global understanding of the class I was trying to sum up what this does in the one public method, and then encapsulate the complexity lower down. When I read this class I want to understand what it does as fast as possible without looking below `private` so tend to prefer this style, but it certainly increases the amount you have to jump around, but hopefully with clearer method names it is clearer overall. Lets go through this class together and see if there is a way to make it clearer.